### ISSUE TITLE
PlantUML fails render map

### ISSUE THREAD

**Issue by rehsack** [docToolchain/docToolchain#588]
**Describe the bug**
PlantUML described how to use object diagrams with maps in [Object Diagram](https://plantuml.com/en/object-diagram). Copying the example into an arc42 project, it fails with:

```
Failed to generate image: PlantUML image generation failed: [From <input> (line 3) ]

@startuml
!pragma graphviz_dot jdot
map CapitalCity {
^^^^^
 Syntax Error?
```

**To Reproduce**
1. Add somewhere into an adoc the map example from plantuml:
```
@startuml
map "Map **Contry => CapitalCity**" as CC {
 UK => London
 USA => Washington
 Germany => Berlin
}
@enduml
```
2. run `doctoolchain . generateHTML generatePDF`
3. **See error**

**Expected behavior**
I expect the example from PlantUML.com is rendered.

**Configuration**
 - OS: Ubuntu 20.04.2
 - Browser n/a
 - Relevant SDK Versions [e.g. JDK 1.8, ...] 
```
java -version
openjdk version "11.0.11" 2021-04-20
OpenJDK Runtime Environment (build 11.0.11+9-Ubuntu-0ubuntu2.20.04)
OpenJDK 64-Bit Server VM (build 11.0.11+9-Ubuntu-0ubuntu2.20.04, mixed mode, sharing)
```

**Additional context**
I tried figuring out how to update asciidoctorj or plantuml - but it seems highly integrated into asciidoctorj and I failed modifying `scripts/AsciiDocBasics.gradle` to use a higher version.

**Find a good headline**
Now it is time to find a good headline/subject for the problem.
Please prefix it with the name of the task which fails.

Example:
exportContributors throws exception when ...

**Comment by rdmueller** [comment]
I guess you are right - we have to upgrade plantUML for maps, yaml and json.
And yes, it seems that we have to upgrade Asciidoc also.

I would like to see this work done in the ng branch for the 2.0 version, but currently, there are more important tasks... :-|

**Comment by mh182** [comment]
This should be fixed now. We `org.asciidoctor.asciidoctorj-diagram` to latest version 2.2.7

**Comment by PacoVK** [comment]
Fixed with #1141  
Tested with `./dtcw generateHTML generatePDF` on commit [8e9e1edbf](https://github.com/docToolchain/docToolchain/commit/8e9e1edbf85a0fb249d7f3fa9e5ba4d3e13594f7) 

Result: 

![image](https://user-images.githubusercontent.com/27785614/235516367-eacefd20-ceee-4e5b-8c1e-4f4d082743aa.png)
